src/style.css

```diff
@@ -1,3 +1,46 @@
+html {
+  font-family:
+    system-ui,
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
+    'Open Sans',
+    'Helvetica Neue',
+    sans-serif;
+}
+
 body {
   margin: 0;
+  display: flex;
+  top: 10px;
+  left: 50px;
+}
+
+.nav {
+  line-height: 15px;
+}
+
+.header {
+  display: flex;
+  padding: 10px 20px;
+}
+
+.logo_img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav_link:hover {
+  width: 37px;
+  height: 4px;
+  top: 56px;
+  left: 509px;
+  border-radius: 8px;
+  margin-left: 40px;
+  background-color: #00acdc;
```

Comment:
on hover to nav link should change color only.
is-active class should be responsible for styles for active link.
links will have different width, so do not hardcode width 37px. set width: 100%